holoviews/element/comparison.py

```diff
@@ -487,8 +487,6 @@ def compare_paths(cls, el1, el2, msg='Path'):
         paths2 = el2.split()
         if len(paths1) != len(paths2):
             raise cls.failureException("%s objects do not have a matching number of paths." % msg)
-        for p1, p2 in zip(paths1, paths2):
-            cls.compare_dataset(p1, p2, '%s data' % msg)
```

Comment:
Should there be an equivalent check or is it ok to remove this comparison?

Reply:
It's already comparing the data so this is just doing it a second time.

Reply:
Really this is more of a test that the interfaces implement split correctly which isn't the job of a comparison.

Reply:
I suppose so though I don't see much harm in this extra check (unless it is really slow or something)

Reply:
Sure, I'll re-enable.